### ISSUE TITLE
i use your pretrained model to test it seems could't work.

### ISSUE THREAD

**Issue by szad670401** [qidiso/mobilefacenet-V2#2]
whatever the pair face image that I input ,the cosine sim always in around 0.60.

**Comment by qidiso** [comment]
do u use the faces_emore database to train? because of  i use the faces_emore database.
released pretrained models  is result of using --loss-type 0 .not is last result use --loss-type 4. 
now i training is not ending. the accurty is increase slowly. if i finished then training ,i will release the last models

**Comment by szad670401** [comment]
i didn't have trained base on you pretrained model. just test. it seems doesn't so well.

**Comment by qidiso** [comment]
i upload a json for softmax .you can try it again
